Assignment3_Spark.py

```diff
@@ -0,0 +1,59 @@
+import sys
+
+from pyspark import StorageLevel
+from pyspark.sql import SparkSession
+
+from Rolling_avg import Rolling_average
+
+
+def main():
+    spark = SparkSession.builder.master("local[*]").getOrCreate()
+
+    database = "baseball"
+    port = "3306"
+    user = " "
+    password = " "
+
+    baseball = (
+        spark.read.format("jdbc")
+        .options(
+            url=f"jdbc:mysql://localhost:{port}/{database}",
+            driver="com.mysql.cj.jdbc.Driver",
+            dbtable="(select Hit,Atbat,batter,b.game_id,DATE(g.local_date) as local_dt \
+                FROM batter_counts b join game g on g.game_id=b.game_id)batters",
+            user=user,
+            password=password,
+        )
+        .load()
+    )
+
+    # Loading the table in a Temp view so that we can reuse it
+    baseball.createOrReplaceTempView("batters")
+    baseball.persist(StorageLevel.DISK_ONLY)
+
+    # Simple SQL
+    batter = spark.sql("(SELECT * FROM batters)")
```

Comment:
There was no need of running the sql.
 baseball.show() should have also displayed the results

Reply:
Could not run the code as sql connector is giving me issues.
Just not sure on how it handles null values. Does it display rolling_avg as null or 0?

Reply:
It displays it as 0. 

Yeah, you are right, that was not required. 
I was just experimenting, this was kinda new to me hence I just tried a simple SQL statement.